app/facades/movies_facade.rb

```diff
@@ -0,0 +1,18 @@
+class MoviesFacade
+  attr_reader :movie_service
```

Comment:
Returns search and top 40, when more than one movie is being displayed. Does not include the level of detail necessary for movie detail page, thus the two facades.